src/reducers/listByIdReducer.ts

```diff
@@ -33,8 +33,9 @@ const listByIdReducer = (state = initialState, action: ListAction): ListState =>
     case FETCH_LIST_START:
       return { ...state, getListStatus: requestStatus.loading };
     case FETCH_LIST_SUCCESS:
+      const itemIds = action.payload.items.map((it: { id: any }) => it.id);
       return {
-        byId: { ...state.byId, [action.payload.id]: action.payload },
+        byId: { ...state.byId, [action.payload.id]: { ...action.payload, itemIds } },
         getListStatus: requestStatus.success,
       };
     case FETCH_LIST_ERROR:
```

Comment:
Let's listen to `CREATE_ITEM_SUCCESS` here. 

from the payload we have to take the list id and push the item id on the `state.byId[listId].itemIds` array.

I will make the necessary change on the backend (we need the list ID on the payload).

Reply:
ok let me know!

Reply:
done! 

new items include `listId`